src/common/localization.js

```diff
@@ -387,24 +387,24 @@ const STRINGS = new LocalizedStrings({
       "Sur la base des commentaires des utilisateurs, nous savons qu'il existe de nombreuses variantes différentes de certains Banis que les gens lisent. Pour vous assurer d'obtenir la meilleure expérience de notre application, veuillez choisir une préférence ci-dessous qui décrit le mieux vos préférences.",
     bani_length_message_2:
       "Ce sera une configuration unique, mais vous pouvez modifier ces paramètres à tout moment en allant dans Paramètres> Longueur de Bani dans l'application.",
-    bani_options: "Options de Bani",
+    BANI_OPTIONS: "Options de Bani",
     baniDB: "BaniDB",
     banidb_living_default: "BaniDB Vivant (défaut)",
     bookmarks: "Signets",
     cancel: "Annuler",
     choose_your_preference: "Choisissez votre préférence",
     click_more_info: "Cliquez ici pour plus d'informations",
-    collect_statistics: "Recueillir des Statistiques",
+    COLLECT_STATISTICS: "Recueillir des Statistiques",
     colored_words: "Mots en Couleurs",
     confirm_exit: "Voulez-vous vraiment quitter?",
     created_by: "Créé par",
     current_saroops_default: "Saroops actuelles (défaut)",
     dark_mode: "Mode nuit",
     default: "Défaut",
     delete: "Supprimer",
-    display_options: "Options d'affichage",
+    DISPLAY_OPTIONS: "Options d'affichage",
     donate: "Faire un don",
-    edit_bani_order: "Modifier la séquence Bani",
+    EDIT_BANI_ORDER: "Modifier la séquence Bani",
```

Comment:
What's the idea behind having few keys in all capitals? Shouldn't these be consistent throughout?